src/main/java/org/assertj/core/api/WithAssertions.java

```diff
@@ -2601,7 +2601,7 @@ default OptionalLongAssert assertThat(final OptionalLong optional) {
    *               assertThat(stats.reboundsPerGame).isBetween(9, 12);
    *            });</code></pre>
    * <p>
-   * {@code assertWith} is variation of {@link AbstractAssert#satisfies(Consumer)} hopefully easier to find for some users.
+   * {@code assertWith} is variation of {@link AbstractAssert#satisfies(Consumer[])} hopefully easier to find for some users.
```

Comment:
I was not sure whether I should add a vararg version to the related method here, since it is an interface:

https://github.com/assertj/assertj-core/blob/016ebe460da0a144f7d24042db4672ee7bc31923/src/main/java/org/assertj/core/api/WithAssertions.java#L2612-L2615

Reply:
I would say yes but it might clash with this one, in this case modify add vararg to this instead of adding a new `assertWith`

Reply:
For consistency I modfied the existing one.